### PR TITLE
Fixes #112

### LINKED ISSUE #112: Active fire popup content issue
The popup for active fires are not showing any content when queried.
 
image

### PR DESCRIPTION
**Pull request by harper-wavra**
Added poly_IncidentName, poly_GISAcres, poly_DateCurrent, irwin_FireCause, irwin_FireCauseGeneral, irwin_FireDiscoveryDateTime, irwin_FireOutDateTime, irwin_UniqueFireIdentifier to `shownFields`.

Updated the active fire perimeters layer to 2021 Wildland Fire Perimeters to date per kitty's request.

### CODE REVIEW

File: src/app/map/map.component.ts [hunk 1]
```diff
@@ -202,13 +202,13 @@ export class MapComponent implements OnInit {
                 if (o.text === "Query by Basin" && o.selected === true) {
                   this.addLayers('NHD Flowlines');
                   this.addLayers('Archived WildFire Perimeters');
```
Comment (amedenblik) [inline src/app/map/map.component.ts:204]: Can you please rename to "Archived Wildfire Perimeters" in the config.json (and elsewhere in the code)?

File: src/assets/config.json [hunk 2]
```diff
@@ -30,8 +30,8 @@
         "visible": false
     },
     {
-        "name": "Active WildFire Perimeters",
-        "url": "https://services3.arcgis.com/T4QMspbfLg3qTGWY/arcgis/rest/services/Current_WildlandFire_Perimeters/FeatureServer/0",
+        "name": "2021 Wildfires Perimeters",
```
Comment (amedenblik) [inline src/assets/config.json:33]: Can you please rename to "2021 Wildfire Perimeters"? Sorry, this was my typo!

**Review by amedenblik — CHANGES_REQUESTED** [review]
Looks good! I have two small comments.

**Comment by harper-wavra** [comment]
I added those new labels to the popups, I was having some trouble, and spending too much time to get them to show up the same way in the report, so I thought we could create another issue and come back to that.

**Comment by amedenblik** [comment]
I'm wondering if we should roll back the new labels in the popups, so that the labels in the popups and report are consistent. Thoughts?

**Comment by harper-wavra** [comment]
Yeah that might be best. It wasnt hard to add it in, so when we have more time to look at the report/html we just add them back.

**Review by amedenblik — APPROVED** [review]

**Pull request merged**